Jenkinsfile

```diff
@@ -284,7 +284,7 @@ def buildHipClangJob(Map conf=[:]){
             }
 
             withDockerContainer(image: image, args: dockerOpts + ' -v=/var/jenkins/:/var/jenkins') {
-                timeout(time: 150, unit:'MINUTES')
+                timeout(time: 600, unit:'MINUTES')
```

Comment:
had to increase this time so CI does not timout.

Reply:
4x is rather dramatic. Do we need that much?

Reply:
I just saw this :) I observed some time outs on gfx908, and I think 210 mins will be sufficient.
10 hours is indeed a bit dramatic